openapi/resource-server.yaml

```diff
@@ -112,7 +112,7 @@ paths:
                 New Incoming Payment for $25:
                   value:
                     id: 'https://openpayments.guide/incoming-payments/08394f02-7b7b-45e2-b645-51d04e7c330c'
-                    walletAddress: 'https://openpayments.guide/alice/'
```

Comment:
I believe `walletAddress` values should still have `/alice`. That seems like the case to me but I could be wrong @mkurapov / @sabineschaller

Reply:
Yeah, the wallet address remains the same, just the baseUrl changes

Reply:
this and similar are now `https://rafiki.money/alice/`